### ISSUE TITLE
Question: Plans for ariadne-graphql-modules

### ISSUE THREAD

**Issue by fabiangfd** [mirumee/ariadne-graphql-modules#19]
Hi, what are the plans for ariadne-graphql-modules going forward? Is it still actively maintained? There seems to be little activity in the repo. Will it eventually become part of the main ariadne project? Just wondering if it makes sense to commit to it now in a project that will run for a long time. Any hints are welcome.

**Comment by rafalp** [comment]
Hi. This repo is not seeing much activity because it pretty much does what it was supposed to do: let people define GraphQL types using SDL, but separate those types into individual Python pieces.

Next big thing for this library is supporting code first approach to types definition, #12. I've did some experiments on this and I've got some ideas for how it's supposed to work, but I've had to pause that because GraphQL proxy tools is the priority for Mirumee.

> Will it eventually become part of the main ariadne project?

Yes. Once it supports both SDL and code first approaches for GraphQL type definitions, it will be rolled into Ariadne. The plan is to let you pick any approach for schema definition that you want, and then migrate your types to other when other one no longer works for you.
